cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ScoreLangConstants.java

```diff
@@ -44,6 +44,15 @@
     String LOOP_KEY = "loop";
     String BREAK_LOOP_KEY = "breakOn";
 
+    // async loop
+    String ASYNC_LOOP_KEY = "async_loop";
+    String ASYNC_LOOP_STATEMENT_KEY = "asyncLoopStatement";
+    String TASK_AGGREGATE_KEY = "taskAggregateValues";
+    String BRANCHES_CONTEXT_KEY = "branches_context";
+    String SPLIT_ITEM_KEY = "splitItem";
```

Comment:
These 3 constants violate the upper condition: BRANCHES_CONTEXT_KEY, SPLIT_ITEM_KEY, BRANCH_RETURN_VALUES_KEY - they are used by runtime so I made an interface for them `io.cloudslang.lang.runtime.RuntimeConstants`. WDYT?